client/stylesheets/_toCat.scss

```diff
@@ -5199,7 +5206,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .headerRadius {
-	width: 70px;
+	width: 150px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, color, margin-left, padding, text-align, width